redirects.json

```
@@ -336,6 +336,62 @@
       "destination": "/guides/status"
     },
 
+    {
+      "name": "React index redirect to site index",
+      "match": "^react/",
+      "destination": "/"
+    },
```

```suggestion
    {
      "name": "Redirect react root",
      "match": "^react/$",
      "destination": "/"
    },
```

Update to match: https://github.com/primer/primer.style/blob/main/redirects.json#L63-L67